subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1702,11 +1710,56 @@ static int twt_args_to_params(const struct shell *sh, size_t argc, char *argv[],
 			params->setup.twt_wake_ahead_duration = (uint32_t)value;
 			break;
 
+		case 'd':
+			if (!parse_number(sh, &value, state->optarg, NULL, 0, 1)) {
+				return -EINVAL;
+			}
+			params->setup.twt_info_disable = (bool)value;
+			break;
+
+		case 'e':
+			if (!parse_number(sh, &value, state->optarg, NULL, 0,
+					  WIFI_MAX_TWT_EXPONENT)) {
+				return -EINVAL;
+			}
+			params->setup.twt_exponent = (uint8_t)value;
+			break;
+
+		case 'm':
+			if (!parse_number(sh, &value, state->optarg, NULL, 0, 0xFFFF)) {
+				return -EINVAL;
+			}
+			params->setup.twt_mantissa = (uint16_t)value;
+			break;
+
 		case 'h':
 			return -ENOEXEC;
 		}
 	}
 
+	if ((params->setup.twt_interval != 0) &&
+	   ((params->setup.twt_exponent != 0) ||
+	   (params->setup.twt_mantissa != 0))) {
+		PR_ERROR("Only set interval or exponent and mantissa\n");
```

Comment:
`Only one of TWT internal or (mantissa, exponent) should be used)`

Reply:
Yes, the implementation select one of twt interval or (mantissa, exponent), will return failed if input both.

Reply:
Updated as comment